panoptes_client/workflow.py

```diff
@@ -179,6 +182,343 @@ def subject_workflow_statuses(self, subject_set_id):
         for status in SubjectWorkflowStatus.where(subject_ids=subject_ids, workflow_id=self.id):
             yield status
 
+    """ CAESAR METHODS """
+
+    def add_to_caesar(self, public_extracts=False, public_reductions=False):
+        """
+        Adds selected Workflow to Caesar. Returns workflow as a dict from Caesar if successful.
+        - **public_extracts** set to True to Enable Public Extracts, Defaults to False
+        - **public_reductions** set to True to Enable Public Reductions. Defaults to False
+
+        Examples::
+            workflow.add_to_caesar()
+            workflow.add_to_caesar(True, True)
+
+        """
+        payload = {
+            'workflow': {
+                'id': self.id,
+                'public_extracts': public_extracts,
+                'public_reductions': public_reductions
+            }
+        }
+        return Caesar().http_post(self._api_slug, json=payload)[0]
+
+    def subject_extracts(self, subject_id):
+        """
+        Returns a list of subject extracts as a dict from Caesar for a given subject.
+
+        Examples::
+            workflow.subject_extracts(1234)
+
+            s = Subject(1234)
+            workflow.subject_extracts(s.id)
+        """
+        url = f'{self._api_slug}/{self.id}/extractors/all/extracts'
+        return Caesar().http_get(url, params={'subject_id': subject_id})[0]
+
+    def subject_reductions(self, subject_id, reducer_key=""):
+        """
+        Returns a list of subject reductions as dicts from Caesar for a given subject.
+        Defaults to return all subject reductions for a given subject.
+        - **reducer_key** If given, will filter and return reductions for the reducer with that reducer_key.
+
+        Examples::
+            workflow.subject_reductions(1234)
+            workflow.subject_reductions(1234,'points')
+        """
+        url = f'{self._api_slug}/{self.id}/subjects/{subject_id}/reductions'
+        if reducer_key and reducer_key.strip():
```

Comment:
Is there a case where `reducer_key` would be true but `reducer_key.strip()` would fail? Is the strip() just to prevent typos, I don't see this on the args to other methods.

Reply:
Ah fair point, this was to prevent `reducer_key` to be a bunch of empty spaces (eg. `reducer_key=".       "`, but I think it is enough to have the `reducer_key.strip()` check. 

I probably should have a `.strip` check on other places, 🤔 .

 I know for some things, I added a `validate_BLAH` method to run before it even makes the request to Caesar. (but that was me trying to mimic the Caesar UI. abit).